ethpm_types/contract_type.py

```diff
@@ -503,14 +504,16 @@ def _get_first_instance(self, _type: Type[ABI_SINGLETON_T]) -> Optional[ABI_SING
         return None
 
     @cached_property
-    def _abi_identifiers(self) -> List[Tuple[ABI_W_SELECTOR_T, str]]:
+    def _abi_identifiers(self) -> list[tuple[ABI_W_SELECTOR_T, str]]:
         def get_id(aitem: ABI_W_SELECTOR_T) -> str:
-            if isinstance(aitem, MethodABI) or isinstance(aitem, ErrorABI):
-                return HexBytes(self._selector_hash_fn(aitem.selector)[:4]).hex()
-            else:
-                return HexBytes(self._selector_hash_fn(aitem.selector)).hex()
+            bytes_val = (
+                self._selector_hash_fn(aitem.selector)[:4]
+                if isinstance(aitem, (MethodABI, ErrorABI))
+                else self._selector_hash_fn(aitem.selector)
```

Comment:
I think of this one as the weird one (basically for `EventABI` it only uses 32 bytes)